### PR TITLE
ci(tests): added unit tets

### PR DESCRIPTION
**Pull request by galta95**
Unit tests in github actions

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -20,11 +19,25 @@ jobs:
...
+        # prettier: true
+
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Run unit tests
+      uses: actions/checkout@v2
+      run: npm run unit-tests
+      with:
+        github_token: ${{ secrets.github_token }}
```
Comment (syncush) [inline .github/workflows/main.yml:43]: the npm script is `test` not `unit-tests`, in addition, it should be a multi-step pipeline 

checkout code -> use node action and build deps -> run the tests

Comment (syncush) [inline .github/workflows/main.yml:43]: in addition checkout@v2 does not need `github_token`

**Review by syncush — CHANGES_REQUESTED** [review]
You should have two separate GitHub actions files, so `lint` and `test` can run concurrently

File: .github/workflows/main.yml (hunk 1 above)
Comment (galta95) [reply .github/workflows/main.yml]: done

Comment (galta95) [reply .github/workflows/main.yml]: done

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+name: Tests
+
+on: [push, pull_request]
```
Comment (syncush) [inline .github/workflows/tests.yml:3]: the trigger should be on `pull_request`

File: .github/workflows/lint.yml [hunk 3]
```diff
@@ -1,30 +1,29 @@
 name: Lint
 
-on: [pull_request]
+on: [push, pull_request]
```
Comment (syncush) [inline .github/workflows/lint.yml:3]: `pull_request`

**Review by syncush — CHANGES_REQUESTED** [review]

**Pull request merged**